File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/idvproperty/d/IdVProperty.java
package iih.ci.ord.idvproperty.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 主键值对象 DTO数据 
 * 
 */
public class IdVProperty extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 系统主键
	 * @return String
	 */
	public String getId() {
		return ((String) getAttrVal("Id"));
	}
	/**
	 * 系统主键
	 * @param Id
	 */
	public void setId(String Id) {
		setAttrVal("Id", Id);
	}
	/**
	 * 系统版本
	 * @return FDateTime
	 */
	public FDateTime getVer() {
		return ((FDateTime) getAttrVal("Ver"));
	}
	/**
	 * 系统版本
	 * @param Ver
	 */
	public void setVer(FDateTime Ver) {
		setAttrVal("Ver", Ver);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/idvproperty/d/IdVProperty.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.idvproperty.d
{
    /// <summary>
    /// IdVProperty 的摘要说明。
    /// </summary>
    public class IdVProperty : BaseDTO {

        public IdVProperty() {
 
        }

        /// <summary>
        /// 系统主键
        /// </summary>
		public string Id {
            get { return getAttrVal<string>("Id",null); }
            set { setAttrVal<string>("Id", value); }
        }

        /// <summary>
        /// 系统版本
        /// </summary>
		public DateTime? Ver {
            get { return getAttrVal<FDateTime>("Ver",null); }
            set { setAttrVal<FDateTime>("Ver", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id", "Ver"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.idvproperty.d.IdVProperty";
        }
    }
}
